RELEASE-NOTES.md

```diff
@@ -0,0 +1,17 @@
+# Release Notes
```

Comment:
Arent release notes usually accompanied with the GitHub release?

Reply:
Like so https://github.com/devfile/api/releases

Reply:
Yeah this would be the case as well, the release section in the release notes file would be added to the release publication on GitHub, this file would just allow us to maintain the release notes towards an upcoming release.

Reply:
This revision I just made the release process will help maintain this: https://github.com/redhat-ai-dev/ai-lab-samples/pull/14#issuecomment-2447822987